tests/test_node.py

```diff
@@ -59,7 +60,7 @@ def test_child_by_field_id(self):
     def test_child_by_field_name(self):
         parser = Parser(self.python)
         tree = parser.parse(b"while a:\n  pass")
-        while_node = tree.root_node.child(0)
+        while_node = cast(Node, tree.root_node.child(0))
         self.assertIsNotNone(while_node)
         self.assertEqual(while_node.type, "while_statement")
         self.assertEqual(while_node.child_by_field_name("body"), while_node.child(3))
```

Comment:
>_possible bug_

**✨ AI Suggestion:** The cast here seems unnecessary as `tree.root_node.child(0)` already returns a `Node` object or None. Consider removing the cast or handling the `None` case explicitly.
```suggestion
        while_node = tree.root_node.child(0)
        self.assertIsNotNone(while_node)
        self.assertEqual(while_node.type, "while_statement")
        self.assertEqual(while_node.child_by_field_name("condition"), while_node.child(1))
```